src/main/java/org/jenkinsCi/plugins/projectDescriptionSetter/DescriptionSetterWrapper.java

```diff
@@ -142,6 +148,11 @@ private String readFile(final FilePath projectDescriptionFile) {
             IOUtils.closeQuietly(in);
             IOUtils.closeQuietly(writer);
             throw new RuntimeException(ioe);
+        } catch (InterruptedException ie) {
+            IOUtils.closeQuietly(reader);
+            IOUtils.closeQuietly(in);
+            IOUtils.closeQuietly(writer);
+            throw new RuntimeException(ie);
```

Comment:
Just add `throws InterruptedException` to the method definition and do something like that with methods using it. `InterruptedException` ideally should not be suppressed at all